src/System.Collections/tests/Performance/System.Collections.Performance.Tests.csproj

```diff
@@ -4,11 +4,12 @@
   <PropertyGroup>
     <IncludePerformanceTests>true</IncludePerformanceTests>
     <ProjectGuid>{16568C86-E97E-42C6-B683-65A1B5AF2EC8}</ProjectGuid>
+    <DisableTests Condition="'$(TargetGroup)' == 'uap' AND ('$(ArchGroup)' == 'arm' OR '$(ArchGroup)' == 'arm64')">true</DisableTests>
```

Comment:
Does this project have a Configurations.props? If not then you won't need Configurations.props